content/go/concepts/comments/comments.md

```diff
@@ -0,0 +1,75 @@
+---
+Title: 'Comments'
+Description: 'A comment is a piece of text within a program that is not executed. It can be used to provide additional information to aid in understanding the code.'
+Subjects:
+  - 'Code Foundations'
+  - 'Computer Science'
+Tags:
+  - 'Comments'
+  - 'Documentation'
+CatalogContent:
+  - 'learn-go'
+  - 'paths/back-end-engineer-career-path'
+  - 'paths/computer-science'
+---
+
+A comment is a piece of text within a program that is not executed. It can be used to provide additional information to aid in understanding the code.
```

Comment:
```suggestion
A **comment** is a piece of text within a program that is not executed. It can be used to provide additional information to aid in understanding the code.
```